docs/salt.md

```diff
@@ -6,11 +6,11 @@ The Salt scripts are shared across multiple hosting providers, so it's important
 
 ## Salt cluster setup
 
-The **salt-master** service runs on the kubernetes-master node.
+The **salt-master** service runs on the kubernetes-master node (except on default GCE setup, see below).
```

Comment:
here's a cool thing: you could have both this and the next parentheses that you added just say "(except on GCE)" and have that text link to `#standalone-salt-configuration-on-gce`, which points to the paragraph you're referencing.